Utils/Functions/System/virtual desktop/script.js

```diff
@@ -0,0 +1,19 @@
+const screenManager = Bean("screenManager");
+
+/**
+ * Obtains the width of user's screen
+ * @returns {number} width of user's screen in pixels
+ */
+// eslint-disable-next-line no-unused-vars
```

Comment:
These `eslint-disable-next-line` before the methods should not be required anymore because the function is now used inside the script, i.e. in the `module.x = x;` call